client/src/app/gui/rtp_packets_table.rs

```diff
@@ -0,0 +1,161 @@
+            .header(30.0, |mut header| {
+                header_labels.iter().for_each(|(label, desc)| {
+                    header.col(|ui| {
+                        ui.heading(label.to_string())
+                            .on_hover_text(desc.to_string());
+                    });
+                });
+            })
+            .body(|body| {
+                self.build_table_body(body);
+            });
+    }
+
+    fn build_table_body(&mut self, body: TableBody) {
+        let mut rtp_packets_ids = Vec::new();
+        self.packets.borrow().iter().for_each(|(&ix, packet)| {
+            if packet.session_protocol == Rtp {
+                rtp_packets_ids.push(ix);
+            }
+        });
+        let packets = self.packets.borrow();
+
+        body.rows(25.0, rtp_packets_ids.len(), |row_ix, mut row| {
+            let first_rtp_id = rtp_packets_ids.first().unwrap();
+            let first_ts = packets.get(first_rtp_id).unwrap().timestamp;
+            let rtp_id = rtp_packets_ids.get(row_ix).unwrap();
+
+            let packet = packets.get(rtp_id).unwrap();
+            let SessionPacket::Rtp(ref rtp_packet) = packet.contents else {
+                panic!("Error. This should be RTP");
+            };
+
+            row.col(|ui| {
+                ui.label(row_ix.to_string());
+            });
+            row.col(|ui| {
+                let timestamp = packet.timestamp - first_ts;
+                ui.label(format!("{:.4}", timestamp.as_secs_f64()));
+            });
+            row.col(|ui| {
+                ui.label(packet.source_addr.to_string());
+            });
+            row.col(|ui| {
+                ui.label(packet.destination_addr.to_string());
+            });
+            row.col(|ui| {
+                ui.label(rtp_packet.version.to_string());
+            });
+
+            row.col(|ui| {
+                ui.label(rtp_packet.padding.to_string());
+            });
+            row.col(|ui| {
+                ui.label(rtp_packet.extension.to_string());
+            });
+
+            row.col(|ui| {
+                ui.label(rtp_packet.marker.to_string());
+            });
+
+            let resp = row.col(|ui| {
+                ui.label(rtp_packet.payload_type.to_string());
+            });
+            resp.1
+                .on_hover_text(PayloadType::new(rtp_packet.payload_type).to_string());
+
+            row.col(|ui| {
+                ui.label(rtp_packet.sequence_number.to_string());
+            });
+
+            row.col(|ui| {
+                ui.label(rtp_packet.timestamp.to_string());
+            });
+
+            row.col(|ui| {
+                ui.label(rtp_packet.ssrc.to_string());
+            });
+
+            row.col(|ui| {
+                if !rtp_packet.csrc.is_empty() {
+                    ui.label(format!("{:?}, ...", rtp_packet.csrc.first().unwrap()))
+                        .on_hover_text(format!("{:?}", rtp_packet.csrc));
+                }
+            });
```

Comment:
I would love if the CSRS were listen line by line.
Instead of `[123,234,234]` do
```
123
234
234
```

Reply:
Formatted